nux/store/actions.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Returns an action object that, when dispatched, presents a guide that takes
+ * the user through a series of tips step by step.
+ *
+ * @param {string[]} tipIDs Which tips to show in the guide.
+ *
+ * @return {Object} Action object.
+ */
+export function triggerGuide( tipIDs ) {
```

Comment:
See bikeshed at #6741 . I think `tipIds` might be most consistent / forward-compatible here.

Reply:
👌 7857e75fa